tests/test_vsitem.py

```diff
@@ -322,8 +2140,8 @@ def test_serialize_datetime(self):
         mock_item = MagicMock(target=VSItem)
 
         b = VSMetadataBuilder(mock_item)
-        b.addMeta({'test_field': datetime(2015,07,12,23,04,31,0,pytz.timezone("Europe/London"))})
-        self.assertEqual(b.as_xml("UTF-8"),"""<?xml version='1.0' encoding='UTF-8'?>
+        b.addMeta({'test_field': datetime(2015,0o7,12,23,0o4,31,0,pytz.timezone("Europe/London"))})
```

Comment:
wouldn't it be easier to simply write these as `7` and `4` rather than `octal-7` and `octal-4`?

Reply:
No idea why it was like that. Changed.